nautobot_netbox_importer/diffsync/base.py

```diff
@@ -0,0 +1,132 @@
+"""Generic DiffSync Importer base module."""
+
+import datetime
+import decimal
+import logging
+from typing import Any
+from typing import Iterable
+from typing import List
+from typing import Mapping
+from typing import MutableMapping
+from typing import Tuple
+from typing import Type
+from typing import Union
+from uuid import NAMESPACE_DNS
+from uuid import UUID
+from uuid import uuid5
```

Comment:
nit: wouldn't the more usual black/isort/ruff convention be to group these, a la:

```suggestion
from typing import (
    Any,
    Iterable,
    List,
    Mapping,
    MutableMapping,
    Tuple,
    Type,
    Union,
)
from uuid import NAMESPACE_DNS, UUID, uuid5
```

Same question/comment throughout.

Reply:
This way passes through all CI checks and produces better git diffs.